content/docs/tech-insights/builtin-data-sources/index.md

```diff
@@ -10,21 +10,219 @@ description: Built-in Data Sources.
 
 Built-in Data Sources use integration configurations from common Backstage plugins and annotations provided by them. You can see configuration instructions for integrations on their respective documentation pages in [the Plugins & Integrations page](/docs/integrations/).
 
-**Annotations used by built-in data sources**
+You will find a list of built-in Data Sources, below, with annotations used by them and facts they currently include.
 
-* **GitHub** and **Dependabot**
-  * `github.com/project-slug`
+### Datadog Data Source
 
-* **PagerDuty**
-  * `pagerduty.com/service-id`
+Imports data from Datadog which can be used to create Tech Insights checks.
 
-* **Datadog**
-  * `datadoghq.com/slo_tag`, for retrieving SLOs 
-  * `datadoghq.com/monitor_tags`, for retrieving monitors
+**Annotations used:** 
+ * `datadoghq.com/slo_tag`, for retrieving SLOs 
+ * `datadoghq.com/monitor_tags`, for retrieving monitors
```

Comment:
Love that we're listing the annotations used. Makes me think we should make this more obvious in the product also.

Reply:
We do have [this one]( https://roadie.io/docs/details/required-annotations/). Do we have customers asking about this often?